src/ops_generator.py

```diff
@@ -6,7 +6,8 @@
 from nodes import NodeState
 import schedule
```

Comment:
@chuanqichen This will need to change drastically.. multiple design flaws here.
After the fix, every node in the system exactly knows when another node dies. Which basically breaks down everything about our project. We'll need to fix this.